go/service/tlf.go

```diff
@@ -60,14 +60,21 @@ func (h *tlfHandler) CompleteAndCanonicalizeTlfName(ctx context.Context, arg key
 	// Append username in case it's not present. We don't need to check if it
 	// exists already since CryptKeys calls below transforms the TLF name into a
 	// canonical one.
-	arg.TlfName = arg.TlfName + "," + string(username)
+	arg.Query.TlfName = arg.Query.TlfName + "," + string(username)
```

Comment:
How does it work with reader TLFs?  

for example: canonical == `patrick,max#chris` and caller sends in `max#chris`, this line would look for `max#chris,patrick` which is different.